src/resources/elements/primeDesignSystem/_styles.scss

```diff
@@ -1,2 +1,6 @@
 @import "./pinput";
 @import "./typography";
+
+.text-center {
```

Comment:
This doesn't always work to center text.  And, is an example of the kind of naming that I want to avoid.